### PR TITLE
add show_install method

### PR DESCRIPTION
**Pull request by tcapelle**
bring back show_install functionalities from V1 to V2

### CODE REVIEW

**Comment by tcapelle** [comment]
Hey Jeremy, I am not sur if you want this methods to be on the test_util file, but I did not wanted to create a new file without your blessing.
Maybe a more general `util.py` file and add all the things that are on the `imports,py` and `torch_core.py` that should be elsewhere?

File: nbs/97_test_utils.ipynb [hunk 1]
```diff
@@ -1,23 +1,13 @@
 {
  "cells": [
-  {
```
Comment (jph00) [inline nbs/97_test_utils.ipynb:3]: Careful not to delete cells like this in your PRs! :) I'll add it back after I merge

Comment (tcapelle) [reply nbs/97_test_utils.ipynb:3]: why it happen? 
I never touched the ipynb's without jupyter...

Comment (jph00) [reply nbs/97_test_utils.ipynb:3]: It means you deleted the cell in Jupyter.

**Pull request merged**